app/controllers/bookings_controller.rb

```diff
@@ -6,8 +6,10 @@ def book
   end
 
   def create
-    book = Booking.new(booking_params)
+    book = Booking.make_new(booking_params)
```

Comment:
Yeah, I didn't know of `before_create` at the time of writing. I will change that tho. Thank you